resources/dicts/patrols/plains/hunting/any.json

```diff
@@ -2409,6 +2409,112 @@
             }
         ]
     },
+    {
+        "patrol_id": "pln_hunt_wabbit1",
+        "biome": ["plains"],
+        "season": ["any"],
+        "types": ["hunting"],
+        "tags": [],
+        "patrol_art": "gen_hunting_rabbit.png",
+        "min_cats": 4,
+        "max_cats": 6,
+        "min_max_status": {},
+        "weight": 5,
+        "intro_text": "It's rabbit season, hahahaha.",
+        "decline_text": "Actually no, would you believe it's duck season?",
+        "chance_of_success": 40,
+        "success_outcomes": [
+            {
+                "text": "Rudely, their first target bolts, and it needs the entire patrol to bring it down, swerving and jolting, zig zagging until cornered. But they <i>do</i> corner it. One rabbit split between so many cats isn't a huge bounty, but they're too tired to continue hunting.",
+                "exp": 20,
+                "weight": 10,
+                "art": "gen_hunt_rabbit.png",
+                "prey": ["small"],
+                "relationships": [
+                    {
+                        "cats_to": ["patrol"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["trust"],
+                        "amount": 3
+                    }
+                ]
+            },
+            {
+                "text": "The rabbits aren't interested in cooperating. Between every cat present, they only manage a single kill. But it's not a failure, and as they pad home, s_c tells the story of El-ahrairah, the Prince with a Thousand Enemies, digger, listener, runner, the prince with the swift warning.",
```

Comment:
WATERSHIP MY BELOVED but also let's make this a tad more subtle.  Maybe just remove the direct name? Keep it to "tells the story of the Prince with a Thousand Enemies, digger, listener, runner, the prince with the swift warning."